nginx/default.conf

```diff
@@ -5,7 +5,7 @@ server {
 	index index.php index.html;
     error_log  /var/log/nginx/error.log;
     access_log /var/log/nginx/access.log;
-	root /var/www/public/;
+	root /var/www/views/;
```

Comment:
public works for me, and its served out of src/public. Can you merge release-v1.0 back into your branch locally, and see if this is still needed?

Reply:
Resolved.